File: LCM/LCM.py
# LCM:
a = int(input("Enter 1st number: "))
b = int(input("Enter 2nd number: "))
if a > b:
    maxNum = a
else:
    maxNum = b
i = maxNum
while i:
    if maxNum % a == 0 and maxNum % b == 0:
        print(f"LCM of {a} and {b} is {maxNum}")
        break
    maxNum += i

File: LCM/LCM.cpp
//  LCM :
#include <iostream>
using namespace std;
int main()
{
    int a, b, i, maxNum;
    cout << "Enter 1st number: ";
    cin >> a;
    cout << "Enter 2nd number: ";
    cin >> b;
    maxNum = a > b ? a : b;
    while (true)
    {
        if (maxNum % a == 0 && maxNum % b == 0)
        {
            cout << "LCM of " << a << " and " << b << " = " << maxNum;
            break;
        }
        maxNum++;
    }
    return 0;
}
